packages/security/__tests__/CredentialManagerFactory.test.ts

```diff
@@ -180,4 +164,31 @@ describe("CredentialManagerFactory", () => {
             expect(CredentialManagerFactory.manager.name).toBe(name);
         });
     });
+
+    describe("determineCredentialManagerType", () => {
+        it("should throw an error if an unsupported credential manager type was passed in as an override", () => {
+            const sampleObject: ICredentialManager = {
+                plugin: "@zowe/cli",
+                type: "notSupportedTypeExample"
+            };
+            expect(() => CredentialManagerFactory.determineCredentialManagerType(sampleObject))
+                .toThrowError("Invalid CredentialManager of type notSupportedTypeExample passed in");
+        });
+        it("should throw an error if an object override with a plugin not of '@zowe/cli' is installed", () => {
+            const sampleObject: ICredentialManager = {
+                plugin: "mySamplePlugin",
+                type: "keytar"
+            };
+            expect(() => CredentialManagerFactory.determineCredentialManagerType(sampleObject))
+                .toThrowError(`Object type credential manager overide is not supported for plugins outside of @zowe/cli`);
```

Comment:
I'm assuming there could be a slight conflict here with how PR #937 is implemented.
It might just be a matter of updating this test once the other PR (#937) is ready to be integrated with this one?